src/index.tsx

```diff
@@ -1,12 +1,26 @@
-import ReactDOM from 'react-dom';
+import { createRoot } from 'react-dom/client';
+import {
+  HashRouter,
+  Route,
+  Routes,
+} from 'react-router-dom';
 
-import './styles/index.css';
-import './styles/todo-list.css';
-import './styles/filters.css';
+import 'bulma/css/bulma.css';
+import './styles/index.scss';
 
 import { App } from './App';
 
-ReactDOM.render(
-  <App />,
-  document.getElementById('root'),
+const Root = () => (
+  <HashRouter>
+    <Routes>
+      <Route path="/" element={<App />}>
+        <Route path="*" element={<p>Page not found</p>} />
```

Comment:
It isn't working correct
<img width="1058" alt="Screenshot 2023-03-08 at 11 24 05" src="https://user-images.githubusercontent.com/53228259/223674196-820076fd-fd5f-4746-b5a3-61219a4bafde.png">